L/liblsl/build_tarballs.jl

```diff
@@ -0,0 +1,82 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "liblsl"
+version = v"1.13.0"
+
+# Collection of sources required to complete build
+sources = [
+    FileSource("https://github.com/sccn/liblsl/archive/1.13.0.tar.gz",
+    "5b304a5365eba33852da96badfbd9d66556caf4a00c87947a59df2942680a617"),
+    DirectorySource("./bundled/patches"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd liblsl-1.13.0
+
+# Add license file
+mkdir  ${prefix}/share/licenses/liblsl
+cp LICENSE ${prefix}/share/licenses/liblsl/
```

Comment:
You can just use
```
install_license LICENSE
```
It'll automatically create the needed directory

Reply:
And do a better job than me forgetting `-p` by the looks of things